yarn.lock

```diff
@@ -2351,6 +2317,8 @@ braces@^2.3.0, braces@^2.3.1, braces@^2.3.2:
     split-string "^3.0.2"
     to-regex "^3.0.1"
 
+<<<<<<< HEAD
+=======
```

Comment:
There is a commit of a git conflict on the package json!

The actual 

```
<<<<<<< HEAD
=======
```

And 

```
>>>>>>> development
```

got COMMITTED into file.  Please, resolve conflicts... do not commit the automatically generated conflict lines.

Reply:
Hey @RafaPolit, this is resolved now